modules/waypoint_tracking.py

```diff
@@ -34,13 +35,13 @@ def get_current_waypoint_info(
     # Get the current destination
     if current_waypoint < drone.commands.count:
         current_command = drone.commands[current_waypoint]
-        if current_command.command == dronekit.mavutil.mavlink.MAV_CMD_NAV_WAYPOINT:
+        if current_command.command == FlightController.dronekit.mavutil.mavlink.MAV_CMD_NAV_WAYPOINT: # not sure if this properly accesses the constant
```

Comment:
you should not access the dronekit like this. flightcontroller contains a function "is_drone_destination_final_waypoint" that has the same logic